project/Build.scala

```diff
@@ -17,17 +17,17 @@ import annotation.tailrec
 
 object SumacBuild extends Build {
   
-  lazy val core = Project("core", file("core"), settings = coreSettings).settings(net.virtualvoid.sbt.graph.Plugin.graphSettings: _*)
-  lazy val ext = Project("ext", file("ext"), settings = extSettings).settings(net.virtualvoid.sbt.graph.Plugin.graphSettings: _*) dependsOn(core)
-  lazy val extZk = Project("ext-zk", file("ext-zk"), settings = extZkSettings).settings(net.virtualvoid.sbt.graph.Plugin.graphSettings: _*) dependsOn(core)
+  lazy val core = Project("core", file("core"), settings = coreSettings)
```

Comment:
So, do the dependency graph settings just not work under 2.12?  Not a big deal either way (I don't think they see much use, and we're likely overdue for an upgrade to the plugin anyway), but it's helpful to have context on why something changed.

Reply:
actually I think I can get this to work if I upgrade the version of sbt and the dependency graph plugin.  then it will be an auto-plugin and still won't need to get mentioned in Build.scala

Reply:
If you're up for taking that on, that would be great.